canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -522,14 +541,16 @@ ToolTip.propTypes = {
 	showToolTipIfTruncated: PropTypes.bool, // Set to true to only display tooltip if full text does not fit in displayable width
 	truncatedRef: PropTypes.object,
 	delay: PropTypes.number,
-	showToolTipOnClick: PropTypes.bool
+	showToolTipOnClick: PropTypes.bool,
+	hoverable: PropTypes.bool, // If true, can hover over to the tooltip, instead of immediately disappearing.
```

Comment:
Sorry picky again. Please change to: "If true, mouse cursor can be hovered over to the tooltip."